src/components/screens/GuideScreen/index.js

```diff
@@ -202,8 +209,22 @@ Guide.propTypes = {
         title: PropTypes.string.isRequired,
       }).isRequired,
     }).isRequired,
-    translationPages: PropTypes.arrayOf(PropTypes.any).isRequired,
+    translationPages: PropTypes.shape({
+      edges: PropTypes.arrayOf(
+        PropTypes.shape({
+          node: PropTypes.shape({
+            fields: PropTypes.shape({
+              language: PropTypes.string,
+              slug: PropTypes.string,
+            }),
+          }),
+        })
+      ),
+    }),
   }).isRequired,
+  pageContext: PropTypes.shape({
+    slug: PropTypes.string,
```

Comment:
Nice! You know I didn't even realize Gatsby offered this data. Good to know! Since it will always be around, can we require it?

```suggestion
    slug: PropTypes.string.isRequired,
```

Reply:
@kylesuss yes it does, when the `createPages` action is running in `gatsby-node.js` whatever is injected via gatsby's special prop `context` can be accessed via `pageContext` in the page, in a nutshell that's how the page queries extract the parameters. I'm going to update the component to reflect the change.